src/views/Auth/Auth.jsx

```diff
@@ -14,6 +14,16 @@ export default function Auth({ isSigningUp = false }) {
     try {
       // TODO: Implement sign up & sign
       // Use isSigningUp to determine whether to sign up or sign in a user
+
+      if(isSigningUp) {
+        const user = await signUpUser(email,password);
+        setUser(user)
+        history.push('/confirm-email')
+      } else {
+        const user = await signInUser(email, password)
+        setUser(user)
```

Comment:
Close! Remember, when signing the user in, you want to set their id and email into the context, not just the user object in full.